### ISSUE TITLE
Avatar on stop local video

### ISSUE THREAD

**Issue by VincentSim** [twilio/video-quickstart-js#50]
Hi all,

```
document.getElementById('stop-video').onclick = function() {
        var localParticipant = room.localParticipant;
         localParticipant.videoTracks.forEach(function (videoTrack) {
            videoTrack.stop();
         });
      };
```
This code works but I want to show an avatar instead of the video stopped for the remote people.
Do you have any idea ?

**Comment by markandrus** [comment]
Hi @VincentSim,

That's good for stopping LocalTracks, but for handling RemoteTracks, I think you'll want to have some &lt;div&gt;—call it `div`—that you're attaching to. Then, when the RemoteTrack is unpublished, you'll want to `detach` from the `div` and replace the contents of `div` with an image. For example,

```js
const div = document.createElement('div');
document.body.appendChild(div); // or insert it elsewhere...

participant.on('trackSubscribed', track => {
  div.appendChild(track.attach());
});

participant.on('trackUnsubscribed', track => {
  track.detach().forEach(element => element.remove());

  const image = document.createElement('image');
  image.src = '/avatar.png';
  image.alt = participant.identity;
  div.appendChild(image);
});
```
